lib/hooks/use-pinch.js

```diff
@@ -1,66 +1,50 @@
-import {
-	useEffect, useCallback, useReducer
-} from 'haunted';
-import {
-	pinchReducer, initState, PINCHING
-} from '../reducers/pinch';
-
-const getDistance = (ev1, ev2) => Math.hypot(ev2.pageX - ev1.pageX, ev2.pageY - ev1.pageY),
-
-	/**
-	 * Tracks the amount a user pinched the document.
-	 *
-	 * Tracking is started by calling the `start` function.
-	 *
-	 * @return {Array} [status: string, delta: number, start: function]
-	 */
-	usePinch = () => {
-		const [state, dispatch] = useReducer(pinchReducer, initState),
-
-			// TODO: handle variable number of touches a la use-touch-pan
-			start = useCallback(ev => {
-				if (ev.touches.length < 2) {
-					return;
-				}
-
-				dispatch({
-					type: 'start',
-					initialDistance: getDistance(...ev.touches)
-				});
-			}, [dispatch]),
-
-			pinch = useCallback(ev => {
-				dispatch({
-					type: 'pinch',
-					distance: getDistance(...ev.touches)
-				});
-			}, [dispatch]),
+import { useEffect } from 'haunted';
+import { pinchReducer } from '../reducers/pinch';
+
+const getDistance = (ev1, ev2) => Math.hypot(ev2.pageX - ev1.pageX, ev2.pageY - ev1.pageY);
+
+/**
+ * Tracks the amount a user pinched the document.
+ *
+ * Tracking is started by calling the `start` function.
+ *
+ * @return {Array} [isPinching: boolean, delta: number, start: function]
+ */
+export const usePinch = () => {
+	const pincher = pinchReducer(),
```

Comment:
Dropping `useReducer` makes the hook not re-render when the state changes.

Reply:
This  `pinchReducer` seems to be a hook actually and not a reducer anymore. If that's the case we should prefix it with `use` ... ?